opcua/ua/uatypes.py

```diff
@@ -1176,4 +1174,52 @@ def __str__(self):
     __repr__ = __str__
 
 
+# TODO: This should be autogeneratd form XML description of EventTypes
+class BaseEvent(FrozenClass):
+
+    '''
+    BaseEvent implements BaseEventType from which inherit all other events and it is used per default.
+
+    '''
 
+    def __init__(self, sourcenode=NodeId(ObjectIds.Server), message=None, severity=1):
```

Comment:
Than BaseEvent is not any more good name :) Since BaseEvent is event type.

OK, I think I understand in which direction you would like to go... to have custom events even there is no that class existing... I am really not fun of that approach since everything will be just more messy, but no problem I can adapt :)